industry_restaurant/data/appointment_type_resource.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="appointment_type_1" model="appointment.type">
+        <field name="resource_ids" model="appointment.resource" eval="[(6, 0, obj().search([('name', '=', 'Floor 1 - Table 1')]).ids)]"/>
```

Comment:
Why don't you take all tables from floor 1 into account?

Reply:
So firstly I have done that only but as per your suggestion that we don't want to show tables on website appointment and directly set the time and the number of bookings directly so for that reason I have removed it

Reply:
But... All ressources should be assigned, no? How would you manage if tables have different capacities?

Reply:
Still.